recipes/libassert/all/conandata.yml

```
@@ -1,5 +1,9 @@
 sources:
   # Newer versions at the top
+  "2.0.0":
+    url:
+      - "https://github.com/jeremy-rifkin/libassert/archive/refs/tags/v2.0.0-beta.tar.gz"
```

```suggestion
    url: "https://github.com/jeremy-rifkin/libassert/archive/refs/tags/v2.0.0.tar.gz"
```

Please, use the correct version. We try to avoid betas in case there is a stable release close to be finished.

---

Thank you for catching this, I had this here as I was trying to iron out the release upstream and I forgot to bump :)